routes/question.js

```diff
@@ -0,0 +1,33 @@
+exports.record = function(req, res){
+	// If the req object does not specify a question or any of the fields
+	// inside the question object, return an error.
+	if(!req.author) {
+		return resError(res, "Sorry, this question doesn't have an author");
+	}
+	else if (!req.timeAsk) {
+		return resError(res, "Sorry, the system is unsure of when the question was asked.");
+	}
+	else if (!req.isStudentQuestion) {
+		return resError(res, "Sorry, this is not a student question.");
+	}
+	else if (!req.session) {
+		return resError(res, "Sorry, this class doesn't have a session.");
+	}
```

Comment:
Should we add in a check for `req.content` too before we use it below in the question creation?